cmd/issueShow.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/xanzy/go-gitlab"
+	"github.com/zaquestion/lab/internal/git"
+	lab "github.com/zaquestion/lab/internal/gitlab"
+)
+
+var issueShowCmd = &cobra.Command{
+	Use:        "show [remote]",
+	ArgAliases: []string{"s", "d"},
```

Comment:
Can you add a regular alias for `get`
```
Aliases: []string{"get"}
```

Also, consider dropping `"d"` as a short alias `"d"` makes me think "delete" over "describe".